src/Sulu/Component/Content/Tests/Unit/Rlp/Strategy/RlpStrategyTest.php

```diff
@@ -256,16 +270,28 @@ public function testLoad()
 
     public function testMove()
     {
-        $contentNode = $this->getMock(NodeInterface::class, [], [], '', false);
-        $contentNode->expects($this->any())
-            ->method('getNodes')
-            ->willReturn([]);
+        $this->mapper->expects($this->any())
+            ->method('loadByContent')
+            ->will($this->returnValue('/test'));
+
+        $document = $this->getMockBuilder(ResourceSegmentBehavior::class)
+            ->getMock();
+        $document->expects($this->any())
+            ->method('getResourceSegment')
+            ->willReturn('/test/test-1');
+
+        $node = $this->getNodeMock();
+
+        $this->documentInspector->expects($this->any())
+            ->method('getNode')
+            ->will($this->returnValue($node));
+
+        $this->isSaved = false;
 
-        $this->isMoved = false;
         // its a delegate
-        $this->strategy->move('/test/test', '/test/test-1', $contentNode, 1, 'default', 'de');
+        $this->strategy->save($document, 1);
 
-        $this->assertTrue($this->isMoved);
+        $this->assertTrue($this->isSaved);
```

Comment:
The `PhpcrMapper`s save method is always called and decides whether to move or save the new resource locator.